crypto/keys/bcrypt/bcrypt.go

```diff
@@ -0,0 +1,291 @@
+// MODIFIED BY TENDERMINT TO EXPOSE `salt` in `GenerateFromPassword`.
```

Comment:
Basically re-add this from the repo as it was some years ago: https://github.com/cosmos/cosmos-sdk/tree/v0.22.0/crypto/keys/bcrypt

Reply:
Confirmed this file is the same as https://github.com/cosmos/cosmos-sdk/blob/v0.22.0/crypto/keys/bcrypt/bcrypt.go with only benign changes.